caravel/templates/index.html

```diff
@@ -3,11 +3,43 @@
 {% block head %}
   {{ super() }}
   <script src="/static/js/continuation.js" type="text/javascript"></script>
+  <script src="/static/js/view.js" type=text/javascript></script>
 {% endblock %}
 {% block content %}
-  <div class="row" id="listings">
-    {% for listing in listings %}{% include "listings/inline.html" %}{%
-      endfor %}
-  </div>
+
+  {% for i in range(2) %}
```

Comment:
lols. You suggested compressing the previous if-else, if -else control flow into for loops. This was what I interpreted the suggestion as.